data/net/minecraft/world/entity/ai/sensing/Sensing.mapping

```diff
@@ -1,3 +1,8 @@
 CLASS net/minecraft/world/entity/ai/sensing/Sensing
+	METHOD <init> (Lnet/minecraft/world/entity/Mob;)V
+		ARG 1 mob
+	METHOD hasLineOfSight (Lnet/minecraft/world/entity/Entity;)Z
+		COMMENT Updates list of visible and not visible entities
```

Comment:
This comment is misleading as it fails to mention that it only checks the given entity.